spec/spec_helper.rb

```diff
@@ -1,8 +1,19 @@
 require "factory_girl_rails"
 require "coveralls"
 require "mocha"
+require 'omniauth'
+require 'capybara/rspec'
+
 Coveralls.wear!
 
+
+module IntegrationSpecHelper
+  def login_with_oauth(service = :github)
+    visit "/auth/#{service}"
```

Comment:
Is this actually going to hit the github api???

If so we can't merge this or else we are going to get throttled by Github after an hour of TDD.